api/MediaSource.json

```diff
@@ -663,50 +670,113 @@
           }
         }
       },
-      "removeSourceBuffer": {
+      "sourceclose_event": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/removeSourceBuffer",
-          "spec_url": "https://w3c.github.io/media-source/#dom-mediasource-removesourcebuffer",
+          "description": "<code>sourceclose</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/sourceclose_event",
+          "spec_url": "https://w3c.github.io/media-source/#dfn-sourceclose",
           "support": {
-            "chrome": {
-              "version_added": "23"
-            },
-            "chrome_android": {
-              "version_added": "25"
-            },
+            "chrome": [
+              {
+                "version_added": "53"
+              },
+              {
+                "version_added": "23",
```

Comment:
OK... so I wanted to confirm at least one of these events, and found my way back to here:
https://chromium.googlesource.com/chromium/src/+/68f43dbea7db4d9632068d7f5a4b8c3774998055

As it turns out, the unprefixed MSE was added separately from the prefixed API, the prefixed API wasn't a mere alias. And the unprefixed events were added only for the unprefixed API.

So actually we should use 31 for these events, as you did originally 😳

Reply:
Cool, thanks for the research! Put all these ranges back to 31 but kept version_removed.